routes/messages/commands/get_summary.js

```diff
@@ -43,17 +51,13 @@ module.exports = async event => {
             repeat_count: progressOnBook.repeat_count + 1
           }
         );
+  } else if (!chatRead && user.credits === 0) {
+    return { text: 'Please, subscribe to read more summaries.' };
```

Comment:
This is perfect, just comment it out so we don't block users from reading books once we merge it.

I assume it was tested locally, did it all work as expected?

Reply:
Yes, it worked.